packages/common/src/CanvasWidget.ts

```diff
@@ -11,9 +12,14 @@ export class CanvasWidget extends HTMLWidget {
         const retVal = super.resize(size);
         this._element.attr("width", this._size.width);
         this._element.attr("height", this._size.height);
```

Comment:
I don't think this can work as there has been no `target` assigned?

Reply:
`resize` isn't called until after there's been a target assigned